server/bot/commands/utilities.ts

```diff
@@ -1,5 +1,5 @@
 // All the types!
-import { Message } from 'eris'
+import Eris, { Message, TextChannel } from 'eris'
```

Comment:
Use GuildTextableChannel, save time fixing merge conflicts with `master`.
```suggestion
import Eris, { Message, GuildTextableChannel } from 'eris'
```